tests/tinytest.R

```diff
@@ -12,7 +12,7 @@ if (requireNamespace("tinytest", quietly=TRUE)) {
     if (packageVersion("tinytest") >= "0.9.3") {
         ## expect_length is in tinytest 1.4.1
         if (!"expect_length" %in% getNamespaceExports("tinytest")) {
-            expect_length <- function(x, n) stopifnot(length(x) == n)
+            expect_length <- function(x, n) expect_equal(length(x), n)
```

Comment:
Good enough for governement work.  There is a bit more in `tinytest::expect_length()` but this may.  Can you actually test this once in a one-off with an ancient `tinytest` ?

Reply:
Good idea -- I'd missed `tinytest` isn't attached. Tests pass.